asio/src/examples/cpp03/CMakeLists.txt

```diff
@@ -0,0 +1,152 @@
+set(target_prefix asio_cpp03_)
```

Comment:
This file is not a proper CML, it's missing `cmake_minimum_required` and `project` calls.